src/main/java/org/opensearch/knn/index/util/KNNLibrary.java

```diff
@@ -68,6 +68,16 @@ public interface KNNLibrary {
      */
     float score(float rawScore, SpaceType spaceType);
 
+    /**
```

Comment:
Cant we re-use the score function for this functionality? Callers should know if they need to translate from distance to score and vice-versa. This function could end up being mis-used. What if lucene adds functionality to go based on distance as well?